packages/Chips/stories/Chips.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import Avatar from "@blaze-react/avatar";
+import { storiesOf } from "@storybook/react";
+import React from "react";
+import ChipsReadme from "../README.md";
+import Chips from "../src";
+
+storiesOf("Chips", module)
+  .addParameters({
+    readme: {
+      sidebar: ChipsReadme
+    }
+  })
+  .add("Introduction", () => (
+    <div className="component-wrapper">
+      <h1>Chips</h1>
+      <Chips modifiers={["deletable", "small"]}>
```

Comment:
deletable and small should be in a constant and if both are accessibles from outside the component do this:

```
const availableModifiers = {
deletable: 'deletable',
small: 'small'

const ChipsComponent = () => <content> bla bla bla</content>

ChipsComponent.availableModifiers = availableModifiers

}

```

And outside the component would be: 
```

<Chips modifiers={[ChipsComponent.availableModifiers.deletable, ChipsComponent.availableModifiers.small]}>
```

This will be very usefull if in a future we have to change the value of the modifiers. Doing that we just have to change this value in one place, the component and we can keeping the key. Also will be easy to use from blaze because we will have autocomplete and would be faster to know that there are this modifiers available.